yarn.lock

```diff
@@ -0,0 +1,604 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
There are a lot of downstream dependencies here. Have they been reviewed for license approval?